packages/ui/src/components/navbar/navbar.tsx

```diff
@@ -1,9 +1,9 @@
 import { useMemo } from 'react'
 import { Link, useLocation, useNavigate } from 'react-router-dom'
 
-import { Button, Icon, NavbarProjectChooser, ScrollArea, Spacer } from '@/components'
+import { Button, Icon, IThemeStore, NavbarProjectChooser, ScrollArea, Spacer } from '@/components'
 import { TypesUser } from '@/types'
-import { TFunction } from 'i18next'
+import { TranslationStore } from '@views/index'
```

Comment:
I had a PR open to allow import from `@components` and `@views`, but it got out of date. Will open a new one, but in the meantime, `@/views` is probably a bit easier. Not a blocker in any way